### PR TITLE
Updated for Django 2.0.1

### PR DESCRIPTION
**Pull request by disestevez**

### CODE REVIEW

File: django_messages/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.db import models, migrations
 from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
```
Comment (arneb) [inline django_messages/migrations/0001_initial.py]: Is this import needed? Can't we just use on_delete=models.CASCADE below?

File: django_messages/templates/django_messages/compose.html [hunk 2]
```diff
@@ -8,7 +8,7 @@ <h1>{% trans "Compose Message"%}</h1>
 <table>
 {{ form.as_table }}
 </table>
-<input type="submit" value="{% trans "Send" %} &raquo;"/>
+<input type="submit" value='{% trans "Send" %} &raquo;'/>
```
Comment (arneb) [inline django_messages/templates/django_messages/compose.html]: This change is not needed. We should keep double quotes for the html attribute.

File: django_messages/templates/django_messages/view.html [hunk 3]
```diff
@@ -28,8 +28,8 @@ <h1>{% trans "Compose reply"%}</h1>
 <table>
 {{ reply_form.as_table }}
 </table>
-<input type="submit" value="{% trans "Send" %} &raquo;"/>
+<input type="submit" value='{% trans "Send" %} &raquo;'/>
```
Comment (arneb) [inline django_messages/templates/django_messages/view.html]: This change is not needed. We should keep double quotes for the html attribute.

File: django_messages/views.py [hunk 4]
```diff
@@ -121,7 +121,7 @@ def reply(request, message_id, form_class=ComposeForm,
         form = form_class(initial={
             'body': quote_helper(parent.sender, parent.body),
             'subject': subject_template % {'subject': parent.subject},
-            'recipient': [parent.sender,]
+            'recipient': [parent.recipient]
```
Comment (arneb) [inline django_messages/views.py]: Are you sure this is a correct change? This is the reply-view, so the user wants to send a reply to the message "parent", so the recipient is the sender of the "parent" message. 
If you disagree, please explain, what's wrong in the original code.

File: django_messages/migrations/0001_initial.py (hunk 1 above)
Comment (imdiegoestevez) [reply django_messages/migrations/0001_initial.py]: Agreed

File: django_messages/templates/django_messages/compose.html (hunk 2 above)
Comment (imdiegoestevez) [reply django_messages/templates/django_messages/compose.html]: Sure. So then you would need to change the double quotes around Send to single. Then: 
`<input type="submit" value="{% trans 'Send' %} &raquo;"/>`
The problem is that in the current version the first quote before Send closes the one of the value attribute, which leads to some sort of chaoes with the remaining 2 quotes.

File: django_messages/views.py (hunk 4 above)
Comment (imdiegoestevez) [reply django_messages/views.py]: Yes you are right, please fix it for me. While I was testing the app, I was writing message to my test account and I wanted to reply to myself. 
Thanks!

File: django_messages/templates/django_messages/compose.html (hunk 2 above)
Comment (arneb) [reply django_messages/templates/django_messages/compose.html]: In fact I think this only breaks Syntax-Highlighting in your editor. The inner part {% trans "Send" %} is evaluated by the Django Template-Engine and the outer part (HTML) which becomes value="Send" is evaluated by your browser. There should be no situation, where the quotes interfere with each other.

Comment (imdiegoestevez) [reply django_messages/templates/django_messages/compose.html]: Oops yeah, it makes a lot sense

**Comment by seospace** [comment]
Is there anything wrong with that branch?

**Comment by leevijakala** [comment]
Hey,
Why not use ModelForm for creating a new message instead of just using forms.Form?
I'm pretty new to django, so I'm just interested to see what I'm missing here...
Thanks.

**Comment by arneb** [comment]
> Why not use ModelForm for creating a new message instead of just using forms.Form?

Take a look at the save method of the ComposeForm, that should answer your question. 
Hint: We are creating one database-entry per recipient instead of only one entry at all.

**Comment by leevijakala** [comment]
Great, got it.
Thanks!

**Comment by arneb** [comment]
thank you, the current code is now compatible

**Pull request closed without merging**